packages/use-segment/src/useSegment.tsx

```diff
@@ -52,34 +54,49 @@ function SegmentProvider<T extends Events>({
   children,
   settings,
   initOptions,
+  areOptionsLoading = false,
```

Comment:
should we check if areOptionLoaded instead of isLoading as with loading state we can have a false, true, false sequency ? 
and with areOptionLoaded this is, false => true  ?

Reply:
I've tested it in integration with the useCookieConsent and it didn't append (because i've passed in the isAnalyticsLoading state that is true by default). But we need to have a default to avoid breaking change no? (and prevent a major release)

Reply:
Maybe I didn't get your point thought.